src/components/ImageGallery/ImageGallery.css

```diff
@@ -0,0 +1,25 @@
+.ImageGallery__Carousel.is-fullscreen {
+  z-index: 30;
+}
+
+.ImageGallery__Carousel.is-fullscreen .ImageGallery__CarouselCell--Container {
```

Comment:
Classname should be `ImageGallery__CarouselCell__Container` as using `--`implies it is a modifier